web/app/helpers/tooltip.js

```diff
@@ -45,10 +44,12 @@ module.exports.showProduction = function(tooltipInstance, mode, country, display
 
     if (displayByEmissions) {
         if (!isStorage) {
-            value *= co2intensity * 1000.0;
+            value *= co2intensity * 1000.0; // MW * gCO2/kWh * 1000 --> gCO2/h
         }
     }
 
+    var absValue = Math.abs(value)
```

Comment:
absValue needed to be defined *after* value has been updated

Reply:
nice catch!!